package.json

```diff
@@ -101,7 +103,8 @@
     "react-router-navigation-prompt": "1.6.3",
     "react-test-renderer": "16.4.0",
     "redux": "4.0.0",
-    "redux-saga": "0.16.0"
+    "redux-saga": "0.16.0",
+    "victory": "32.3.1"
```

Comment:
Is this still needed with the newer version of `@patternfly/react-charts`?  Maybe just a refresh of `yarn.lock` will allow removing the explicit dependency.  That would be nice since we're not using it directly.